opsdroid/core.py

```diff
@@ -369,7 +369,7 @@ def _connector_names(self):  # noqa: D401
                     and cls is not Database
                 ):
                     _LOGGER.debug(_("Adding database: %s."), name)
-                    database = cls(database_module["config"])
+                    database = cls(database_module["config"], opsdroid=self)
```

Comment:
@jacobtomlinson I assume this is correct, it seems a bunch of the databases expected this?

Reply:
Yeah probably.